packages/atlas/src/views/viewer/DiscoverView/DiscoverView.tsx

```diff
@@ -29,9 +27,9 @@ export const DiscoverView: FC = () => {
 
   const featuredVideoCategoryCardsData = useMemo(() => {
     const _featuredVideoCategoryCardsData =
-      categories
-        ?.map((category) => {
-          const video = categoriesFeaturedVideos?.[category.id]?.categoryFeaturedVideos.find(
+      displayCategoriesWithCounter
+        .map((category) => {
+          const video = categoriesFeaturedVideos?.[category.defaultVideoCategory]?.categoryFeaturedVideos.find(
```

Comment:
This isn't correct - like I mentioned previously, video featuring will not use QN categories but display categories IDs. So to get featured videos for a display category you should just do `categoriesFeaturedVideos[displayCategory.id]`